### DISCUSSION TITLE
Keyball39 の左手の基板(トラックボールがない方)のみ購入は可能でしょうか

### DISCUSSION THREAD

**Discussion by shiro30330** [Yowkees/keyball#116] [Q&A] [answered]
お世話になっております、https://github.com/Yowkees/keyball/discussions/115 でトラックボール読み取り基板を購入した者です。

片手だけキー入力できない問題にぶち当たり [こちら](https://yushakobo.zendesk.com/hc/ja/articles/360043587373-%E5%88%86%E5%89%B2%E3%82%AD%E3%83%BC%E3%83%9C%E3%83%BC%E3%83%89%E3%81%AE%E7%89%87%E6%89%8B%E3%81%A0%E3%81%91%E5%85%A5%E5%8A%9B%E3%81%A7%E3%81%8D%E3%81%AA%E3%81%84) を参考に TRRS ジャックのはんだ付けをやり直そうとしたのですが、無理に外そうとしてしまったため基板の穴についていた銅色の膜まで剥がれてしまいました。

2 点質問させてください。

1. 銅色の膜まで剥がれてしまったらもう基板を変えるしかないでしょうか。
1. Keyball 46 のように保守用で Keyball 39 の基板の販売はございますでしょうか。

大変恐れ入りますがご回答いただけると幸いです。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
了解しました。トラックボール有り側の基板と、それに付随するPCBソケットやダイオード等部品を特別価格にて販売させていただくのはどうでしょうか？
ここから先のやり取りをメールかツイッターのDMでさせていただきたく、お教えいただければ幸いです🙏

**Comment by Yowkees** [comment]
あっそれか、リカバリできるかも知れませんのでダメージ部分の写真を見せていただければ🙏

**Comment by shiro30330** [comment]
ご返事ありがとうございます。
現状の基板と TRRS ジャックに張り付いたまま剥ぎ取ってしまった銅色の膜の写真を添付いたします。
![IMG_4944](https://user-images.githubusercontent.com/104175523/165088747-4fbf3b1f-ff44-429f-ba21-587b354e9a23.jpg)
![IMG_4946](https://user-images.githubusercontent.com/104175523/165088786-22247b4f-f2ce-4e76-9b12-872713b59f28.jpg)
![IMG_4947](https://user-images.githubusercontent.com/104175523/165088807-ac7481c2-aac1-4401-a963-580bc1a879ef.jpg)
![IMG_4948](https://user-images.githubusercontent.com/104175523/165088817-ea334e15-fa9e-4e96-a177-8390313832ff.jpg)

> 了解しました。トラックボール有り側の基板と、それに付随するPCBソケットやダイオード等部品を特別価格にて販売させていただくのはどうでしょうか？

ありがとうございます、リカバリが難しかったら購入させていただきたいです。
Twitter のアカウントは @shiro30330 です、よろしくおねがいします。

**Reply by shiro30330** [reply]
> トラックボール有り側の基板と、

あ、すみません、基板はトラックボールがない方の基板が希望です！

**Reply by Yowkees** [reply]
銅箔が完全に剥離したのですね。修理は不可能ではないですが、抜き差しで力がかかる部品なので使い勝手がわるくなりそうです。
トラックボール無し側で了解しました。
DMでご連絡させていただきます🙏

**Reply by shiro30330** [reply]
ご確認いただきありがとうございます、いただいた DM に返信いたしました 🙏

**Reply by shiro30330** [reply]
届いた新しい基板を組み立てて左右ともキー入力できるようになりました。
Yowkees さんの手厚いフォローのおかげです。
まだ Keyball 39 の入力に慣れていないのですがこれから使い倒そうと思います。
ありがとうございました！

**Reply by Yowkees** [reply]
無事に完成したようで良かったです！ぜひ使ってあげてください🙏
